### DISCUSSION TITLE
i want the xrdp gui

### DISCUSSION THREAD

**Discussion by CsehRafin** [linuxdeepin/developer-center#3181] [Features Request & Ideas | 特性请求 & 头脑风暴]
i want like this :
![download](https://user-images.githubusercontent.com/94455536/170280251-a9c58c0c-98b1-466f-a7f8-4da448766cb9.jpg)
with xrdp like it works on kali linux and ubuntu 
and yes i have systemd on deepin (the deepin roofs that you released)
i got the systemd with this :(https://github.com/nullpo-head/wsl-distrod)
i want the GUI with the dde

**Comment by BLumia** [comment]
The image is too small, could you provide a larger one so I can know what exactly do you want? Thanks!

**Comment by kt286** [comment]
you can try this

https://github.com/gfdgd-xi/simple-remote-desktop-accessor

**Comment by CsehRafin** [comment]
I have installed deepin in my wsl version 20.5 but I want the GUI with dde not xfce or gnome or KDE
